stable/mysqldump/templates/configmap.yaml

```diff
@@ -10,6 +10,16 @@ data:
   MYSQL_PORT: {{ .Values.mysql.port | quote }}
   MYSQL_OPTS: {{ .Values.options | quote }}
   KEEP_DAYS: {{ .Values.housekeeping.keepDays | quote }}
+{{- if .Values.upload.openstack.enabled }}
+  OS_AUTH_URL: {{ .Values.upload.openstack.authUrl }}
+  OS_PROJECT_NAME: {{ .Values.upload.openstack.project }}
+  OS_PROJECT_DOMAIN_NAME: {{ .Values.upload.openstack.projectDomain }}
+  OS_USERNAME: {{ .Values.upload.openstack.user }}
+  OS_USER_DOMAIN_NAME: {{ .Values.upload.openstack.userDomain }}
+{{- if not .Values.upload.openstack.existingSecret }}
+  OS_PASSWORD: {{ .Values.upload.openstack.password }}
```

Comment:
this should be referenced from a secret.

Reply:
True. Thanks for spotting it.